### PR TITLE
WRO-11485: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies for Moonstone.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Dependencies now use the latest version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-11485

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -44,26 +44,26 @@
...
     "warning": "^4.0.3",
-    "ilib": "14.14.0"
+    "ilib": "^14.15.1"
   },
   "peerDependencies": {
-    "ilib": "14.14.0 || ^14.14.0-webos1"
+    "ilib": "^14.15.1 || ^14.15.1-webos1"
   },
   "devDependencies": {
     "@enact/docs-utils": "0.3.6",
```
Comment (hong6316) [inline package.json]:
```suggestion
    "@enact/docs-utils": "^0.4.0",
```

File: package.json [hunk 2]
```diff
@@ -44,26 +44,26 @@
...
     "@enact/ui-test-utils": "^1.0.0",
```
Comment (hong6316) [inline package.json]:
```suggestion
    "@enact/ui-test-utils": "^1.0.1",
```

**Comment by hong6316** [comment]
- Please update `package.json` and `npm-shrinkwrap.json` in agate/samples/sampler, too.
- Let's restore `package-lock.json` in the root directory. (Please do not revert but recreate one(npm install))

**Comment by andreitirlalge** [comment]
> * Please update `package.json` and `npm-shrinkwrap.json` in agate/samples/sampler, too.
> * Let's restore `package-lock.json` in the root directory. (Please do not revert but recreate one(npm install))

I have managed to add root package-lock.json and update sampler dependencies but regarding all @storybook dependencies, they cannot be updated at the moment because some several webpack configurations must be made. They were handled in this PR https://github.com/enactjs/moonstone/pull/141

**Review by hong6316 — APPROVED** [review]
LGTM

**Review by hong6316 — APPROVED** [review]
LGTM

File: package.json [hunk 3]
```diff
@@ -44,26 +44,26 @@
...
+    "@enact/ui": "^4.5.2",
+    "classnames": "^2.3.2",
     "invariant": "^2.2.4",
     "prop-types": "^15.8.1",
     "ramda": "^0.28.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "warning": "^4.0.3",
-    "ilib": "14.14.0"
+    "ilib": "^14.15.1"
```
Comment (MikyungKim) [inline package.json:58]: As I commented on agate PR, please pin this version to 14.14.0.

Comment (andreitirlalge) [reply package.json:58]: Hello @MikyungKim , as Daniel mentioned in https://github.com/enactjs/agate/pull/687. Ilib team actually released 14.15.1 a few days after enact team created an issue. The isomorphic build failure issue is fixed https://github.com/iLib-js/iLib/releases/

**Review by daniel-stoian-lgp — APPROVED** [review]

File: samples/sampler/package.json [hunk 4]
```diff
@@ -20,13 +20,13 @@
...
+    "@enact/i18n": "^4.5.2",
     "@enact/moonstone": "../../",
-    "@enact/spotlight": "^4.5.0",
-    "@enact/ui": "^4.5.0",
-    "classnames": "^2.3.1",
-    "ilib": "14.14.0",
+    "@enact/spotlight": "^4.5.2",
+    "@enact/ui": "^4.5.2",
+    "classnames": "^2.3.2",
+    "ilib": "14.15.1",
```
Comment (MikyungKim) [inline samples/sampler/package.json]:
```suggestion
    "ilib": "^14.15.1",
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**